cmd/config.go

```diff
@@ -0,0 +1,75 @@
+package cmd
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+
+	guid "github.com/google/uuid"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+// configCmd represents the config command
+var configCmd = &cobra.Command{
+	Use:   "config",
+	Short: "Links uuid with Snyk token.",
+	Long: `config registers Snyk Token 
+	with UUID on crda server and outputs UUID.`,
+	Args: validateArgs,
+	Run:  main,
+}
+
+func init() {
+	rootCmd.AddCommand(configCmd)
+
+}
+func validateArgs(cmd *cobra.Command, args []string) error {
+	if len(args) < 1 {
+		return errors.New("snyk token is missing")
+	}
+	return nil
+}
+
+func requestServer(synkToken string) string {
+	ThreeScale := viper.GetString("3scaleToken")
+	url := viper.GetString("server") + "/user?user_key=" + string(ThreeScale)
+	uuid := guid.New().String()
+	payload, err := json.Marshal(map[string]string{
+		"snyk_api_token": synkToken,
+		"user_id":        uuid,
+	})
+
+	client := &http.Client{}
+	req, err := http.NewRequest(http.MethodPut, url, bytes.NewBuffer(payload))
+
+	if err != nil {
+		fmt.Println(err)
+		panic("Unable to build request")
+	}
+	req.Header.Add("Content-Type", "application/json")
+
+	res, err := client.Do(req)
+	if err != nil {
+		fmt.Println(err)
+		panic("Unable to request server")
+	}
+	defer res.Body.Close()
+	if int(res.StatusCode) != 200 {
+		fmt.Println(err)
+		panic("Server Errored. Please try again.")
+	}
+	return uuid
```

Comment:
Are you not supposed to store this in the filesystem

Reply:
I am of the opinion that Admin will pick that token and push that in Jenkins env.
I think that's what we discussed here...
https://docs.google.com/document/d/18A7YGdN7MsL6S39aQBnFF1djX93oU5foPlZDSHIiQms/edit?ts=5fbf80bc#heading=h.1g50olz5mv0j 

Please check, how it will look like in the attached image..
![Annotate_a_local_image(2)](https://user-images.githubusercontent.com/21026016/100463421-ce389b80-30f1-11eb-9851-e8d735f34b18.png)

Reply:
Thats fine but what about the case when it used as plain CLI. We decided that the UUID will be stored in the filesystem or an environment variable.

Reply:
+1

Also should have a flag to show the token on the stdout? something like `--show-token`?

Reply:
Added UUID in env. Updated name to `CRDA_KEY` to avoid conflicts with any existing env variables in the user's system.

Reply:
> Also should have a flag to show the token on the stdout? something like --show-token? 

Ack. Flag added as `--show` `-s`